### ISSUE TITLE
Add questions for accessibility concerns on the part of attendees

### ISSUE THREAD

**Issue by melinath** [littleweaver/django-brambling#306]
We should have a section of questions regarding accessibility - for example, so that events can hire ASL interpreters or make extra sure their venues are wheelchair-accessible.
